tests/src/main/java/io/openliberty/tools/eclipse/test/it/AbstractLibertyPluginSWTBotTest.java

```diff
@@ -122,29 +122,29 @@ protected static void cleanupProject(String projectPathStr) {
      * @throws InterruptedException
      * @throws CoreException
      */
-    public static void importMavenProjects(File workspaceRoot, List<String> folders) {
-
-        Display.getDefault().syncExec(new Runnable() {
-
-            @Override
-            public void run() {
-                try {
-                    // Get the list of projects to install.
-                    MavenModelManager modelManager = MavenPlugin.getMavenModelManager();
-                    LocalProjectScanner lps = new LocalProjectScanner(folders, false, modelManager);
-                    lps.run(new NullProgressMonitor());
-                    List<MavenProjectInfo> projects = lps.getProjects();
-
-                    // Import the projects.
-                    ProjectImportConfiguration projectImportConfig = new ProjectImportConfiguration();
-                    IProjectConfigurationManager projectConfigurationManager = MavenPlugin.getProjectConfigurationManager();
-                    projectConfigurationManager.importProjects(projects, projectImportConfig, new NullProgressMonitor());
-                } catch (Exception e) {
-                    e.printStackTrace();
-                }
-            }
+    public static void importMavenProjects(File workspaceRoot, List<String> folders) throws Exception {
+
+        // Get the list of projects to install.
+        MavenModelManager modelManager = MavenPlugin.getMavenModelManager();
+        
+        for (String folder : folders) {
```

Comment:
I was just grasping and wondering if it would be more or less likely to hit something like the #375 issue.  

It did not seem to eliminate some of the error messages I noted above so far...